pkg/controller/globalrolebinding/globalrolebinding_controller.go

```diff
@@ -213,9 +212,7 @@ func (c *Controller) reconcile(key string) error {
 		return err
 	}
 
-	isPlatformAdmin := globalRoleBinding.RoleRef.Name == iamv1alpha2.PlatformAdmin
-
-	if isPlatformAdmin {
+	if isPlatformAdmin := globalRoleBinding.RoleRef.Name == iamv1alpha2.PlatformAdmin; isPlatformAdmin {
```

Comment:
If `isPlatformAdmin` isn't used anywhere else, why bother to create one?
```suggestion
	if globalRoleBinding.RoleRef.Name == iamv1alpha2.PlatformAdmin {
```